### DISCUSSION TITLE
VRS 2.0 Allele RLE representation on VCF annotation

### DISCUSSION THREAD

**Discussion by ahwagner** [ga4gh/vrs-python#408] [Ideas]
VRS 2.0 alleles can normalize to Reference Length Expressions. We should discuss what this looks like for our VCF annotator.

**Comment by theferrit32** [comment]
I've implemented this in PR #589 and this substitution variant (ref included) is an example used in tests there:
```
chr19	82664	.	C	T	50	PASS	VRS_Allele_IDs=ga4gh:VA.esmv8nARRRdSysDFuIErJxRAdUSVsWNE,ga4gh:VA.b9AIUegs1SOYO7nKY8l01yllkyUns-u4;VRS_Starts=82663,82663;VRS_Ends=82664,82664;VRS_States=C,T;VRS_Lengths=1,.;VRS_RepeatSubunitLengths=1,.	GT:PS:DP:ADALL:AD:GQ	0/1:.:154:0,0:0,0:120
```
https://github.com/ga4gh/vrs-python/blob/26eeba3e99f065e2db331b48d62f56e9a6647eef/tests/extras/data/test_vcf_expected_output.vcf#L239C1-L240C1


And for some proper ReferenceLengthExpressions in repeating regions:
https://github.com/ga4gh/vrs-python/blob/26eeba3e99f065e2db331b48d62f56e9a6647eef/tests/extras/data/test_rle.vcf

The expected values are checked here rather than in an annotated vcf file checked into git:
https://github.com/ga4gh/vrs-python/blob/26eeba3e99f065e2db331b48d62f56e9a6647eef/tests/extras/test_annotate_vcf.py#L263-L311
